content/coordination/onboarding/index.en.md

```diff
@@ -60,24 +60,24 @@ For alignment with the R-Ladies mission can you just write an ok as an answer to
 
         -   Country
 
-        -   Organisers
+        -   Organizers
 
         -   Status = Prospective
 
--   Invite organisers to the organisers slack.
+-   Invite organizers to the organizers slack.
 
--   Send the prospective organiser an email with all the most important
+-   Send the prospective organizer an email with all the most important
```

Comment:
"organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.

Reply:
```suggestion
-   Send the prospective organiser an email with all the most important
```